Cython/Compiler/Nodes.py

```diff
@@ -1044,7 +1044,10 @@ def analyse(self, env, could_be_name=False):
                 scope = env
                 for item in self.module_path:
                     entry = scope.lookup(item)
-                    if entry is not None and entry.is_cpp_class:
+                    if entry is not None and (
```

Comment:
Is there ever a situation where this might need a while loop (a typedef of a typedef)?

This looks like the right approach though

Reply:
Good call. Let me check that.